### DISCUSSION TITLE
Welcome to supabase-unity-template Discussions!

### DISCUSSION THREAD

**Discussion by wiverson** [wiverson/supabase-unity-template#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gizmotronn** [comment]
Hi @wiverson , I was wondering if there were any templates or examples around showing data from supabase tables in Unity scenes? My team and I are wanting to use Supabase as the inventory system for our Unity game but there's not a lot of clarity around pulling & pushing data via the current unity/c# library

**Reply by wiverson** [reply]
The API for the C# library is based on the JavaScript API, so a lot of docs for that portion of things should be similar.

From a high level, many folks use the library to build queries against public table data and use RLS to secure the data.

I prefer to just keep my data in a private app schema and then put stored procedures in public. I find that easier to manage.

At the end of the day it's Postgres with some extra services. The library is mainly a usability shim.

**Reply by wiverson** [reply]
Have you used Postgres/a relational db before?

**Reply by Gizmotronn** [reply]
Have only used postgres with Supabase, but have used it extensively with Python/Flask & Typescript/Nextjs
Is there a sample script or folder/repo you can point me towards that shows how c# can be used to pull data from a table? In theory I assume that you could repurpose a c#/dotnet script that does this into a Unity script -> however I'm unable to get my head around setting up my local environment for the initial scripting

Thanks for the reply
